ui/src/app/views/homepage/component.ts

```diff
@@ -66,9 +75,7 @@ export class HomepageComponent implements OnInit, OnDestroy {
           this.profileStorageService.reload();
         }, 10000);
       }
-      if (profile != null) {
-        this.FCAccountInitialized = true;
-      }
+      this.FCAccountInitialized = true;
```

Comment:
Can we just remove this variable entirely and rely only on hasCDRAccess? Basically, as it stands right now, when will this ever be false?

Reply:
It's going to be false when the profile hasn't loaded yet. The home page does render before that. We still need error messaging to look at this - I switched the template to use `hasCdrAccess` as the source of truth for the list workspace page, but had to include the profile for the error messaging cases.

Reply:
Got it! I didn't notice that this was within that subscribe. LGTM.